### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by KINGOFKINGOFKEY** [mkdocstrings/mkdocstrings#717] [Documentation]
I'm trying to access a character in a string, but I keep getting an `IndexError` that says "string index out of range." How can I make sure that the index I'm using is within the valid range of the string?

**Comment by pawamoy** [comment]
Sorry again, it seems like you're looking for help with Python coding, and not about anything specific to mkdocstrings? You'll have more luck getting help on [Python Discord](https://discord.com/invite/python), or [Python Discourse forum](https://discuss.python.org). I'll close this :relaxed:
